packages/vkui/src/components/Alert/Alert.tsx

```diff
@@ -139,6 +146,14 @@ export const Alert = ({
     [close],
   );
 
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    if (!allowClickPropagation) {
+      stopPropagation(event);
+    }
+  };
+
+  const clickHandlers = mergeCalls({ onClick: handleClick }, { onClick });
```

Comment:
```suggestion
  const handleClick = allowClickPropagation ? onClick : (event: React.MouseEvent<HTMLElement>) => {
    stopPropagation(event);
    onClick?.(event)
  };
```
не очень импонирует `mergeCalls` – внутри создается массив, потом внутри объект трасформируется в массив и потом итерируем по этому массиву

в данном случае, выглядит так, что проще не оборачивать в наш обработчик если передан `allowClickPropagation`

---

тоже самое в `ActionSheet` и его производных